loony_dev/tests/test_supervisor_remote_control.py

```
@@ -0,0 +1,322 @@
+                log_file=self.log_file,
+                pid_file=self.pid_file,
+                conn_file=self.conn_file,
+            )
+        return rcp, ctx
+
+    def test_builds_expected_command_and_cwd(self) -> None:
+        rcp, ctx = self._launch()
+        # The spawned process targets the child entrypoint with the right args.
+        proc = ctx.created[0]
+        self.assertIs(proc.target, supervisor._run_remote_control_process)
+        log_file, conn_file, repo, base_dir, session_id, key, _started = proc.args
```

Ruff (0.15.15)

[warning] Unpacked variable `log_file` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)